x-pack/plugins/security_solution/public/resolver/view/graph_controls.test.tsx

```diff
@@ -0,0 +1,221 @@
+    });
+
+    it('should show the origin node further left on the screen', async () => {
+      await expect(originNodeStyle()).toYieldObjectEqualTo({
+        left: '796.93132px',
+        top: '535.5792px',
+      });
+    });
+  });
+
+  describe('when the user clicks the south panning button', () => {
+    beforeEach(async () => {
+      (await simulator.resolve('resolver:graph-controls:south-button'))!.simulate('click');
+      simulator.runAnimationFramesTimeFromNow(nudgeAnimationDuration);
+    });
+
+    it('should show the origin node lower on the screen', async () => {
+      await expect(originNodeStyle()).toYieldObjectEqualTo({
+        left: '746.93132px',
+        top: '485.5792px',
+      });
+    });
+  });
+
+  describe('when the user clicks the east panning button', () => {
+    beforeEach(async () => {
+      (await simulator.resolve('resolver:graph-controls:east-button'))!.simulate('click');
+      simulator.runAnimationFramesTimeFromNow(nudgeAnimationDuration);
+    });
+
+    it('should show the origin node further right on the screen', async () => {
+      await expect(originNodeStyle()).toYieldObjectEqualTo({
+        left: '696.93132px',
+        top: '535.5792px',
+      });
+    });
+  });
+
+  describe('when the user clicks the north panning button', () => {
+    beforeEach(async () => {
+      (await simulator.resolve('resolver:graph-controls:north-button'))!.simulate('click');
+      simulator.runAnimationFramesTimeFromNow(nudgeAnimationDuration);
+    });
+
+    it('should show the origin node higher on the screen', async () => {
+      await expect(originNodeStyle()).toYieldObjectEqualTo({
+        left: '746.93132px',
+        top: '585.5792px',
+      });
+    });
+  });
+
+  describe('when the user clicks the center panning button', () => {
+    beforeEach(async () => {
+      (await simulator.resolve('resolver:graph-controls:north-button'))!.simulate('click');
+      simulator.runAnimationFramesTimeFromNow(nudgeAnimationDuration);
+      (await simulator.resolve('resolver:graph-controls:center-button'))!.simulate('click');
+      simulator.runAnimationFramesTimeFromNow(nudgeAnimationDuration);
+    });
+
+    it("should return the origin node to it's original position", async () => {
+      await expect(originNodeStyle()).toYieldObjectEqualTo(originalPositionStyle);
+    });
+  });
+
+  it('should show the origin node as larger on the screen', async () => {
+    await expect(originNodeStyle()).toYieldObjectEqualTo(originalSizeStyle);
+  });
+
+  describe('when the zoom in button is clicked', () => {
+    beforeEach(async () => {
+      (await simulator.resolve('resolver:graph-controls:zoom-in'))!.simulate('click');
+      simulator.runAnimationFramesTimeFromNow(nudgeAnimationDuration);
+    });
+
+    it('should show the origin node as larger on the screen', async () => {
+      await expect(originNodeStyle()).toYieldObjectEqualTo({
+        width: '427.7538290724795px',
```

Comment:
:information_source: this comes from the yarn snapshot deal?

Reply:
yep